energy-integration/farm-boosted-yields/src/lib.rs

```diff
@@ -30,6 +30,16 @@ impl<M: ManagedTypeApi> SplitReward<M> {
     }
 }
 
+mod energy_factory_proxy_send_rew {
```

Comment:
Why not import the actual energy factory code? This way we can also do the integration tests. Si I would replace this with the actual reference.